### DISCUSSION TITLE
Can I limit GlazeWM to only one monitor?

### DISCUSSION THREAD

**Discussion by fakebizprez** [glzr-io/glazewm#753] [Q&A]
I have a triple monitor setup.
I run a 34-inch wide screen in the center with two 32-inch monitors on the side.
If possible, how can I modify my GlazeWM configuration to only be enabled on the center monitor?
If not possible, I'd be happy to hear any alternative ideas.

Thank you.

![image](https://github.com/user-attachments/assets/0215f24f-2747-449b-8d5d-47410a83d2bb)

**Comment by LJFRIESE** [comment]
Did you find any solutions to this? I'm in a similar situation.

**Comment by ianklapouch** [comment]
Same issue here

**Comment by Alxlol** [comment]
Curious about this aswell

**Comment by neilyhy** [comment]
`bind_to_monitor: 1` on every workspaces?

**Comment by woncomp** [comment]
Similar request here, I only want glazeWM to manage my vertical monitor on the side, while keeping my center monitor freeform.
